packages/google_maps_flutter/google_maps_flutter/example/ios/Runner/AppDelegate.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
```

Comment:
remove

Reply:
I will revert the `example`. I think `flutter create` was run inside the `example` dir, and it added a lot of stuff that is not needed for this PR.